Copy-pasta static HTML for Personal Information.

awong-dev opened the pull request:
This provides a baseline to work off of.

awong-dev committed:
Copy-pasta static HTML for Personal Information.

This provides a baseline to work off of.

akainic commented on _health-care/_js/_components/personal-information-panel.jsx:
```diff
@@ -1,44 +1,26 @@
+import VeteranAddressSection from './veteran-address-section';
 
 class PersonalInformationPanel extends React.Component {
-  constructor() {
```
Wait, we can delete this?

akainic commented on _health-care/_js/_components/name-and-general-info-section.jsx:
```diff
@@ -0,0 +1,138 @@
+            <h4>Place of Birth</h4>
+            <div>
+              <div className="usa-input-grid usa-input-grid-medium">
+                <label htmlFor="veteran_city_of_birth">City</label>
```
We'll need to make sure all the labels and inputs are associated eventually, I can create a Waffle ticket to check those

awong-dev replied:
Yes. I know these are all broken. I actually think as we go to create a <ValidatedInput> component that will handle all this for us.

akainic commented:
Should we eventually create subdirectories in the `_components/` directory to organize some of the components by topic?

awong-dev commented:
Yes...but maybe after I finish this?

awong-dev committed:
Copy-pasta HTML for Insurance & Military Info.

awong-dev committed:
Copy-pasta rest of static HTML sections.

After this, form-react should have all the fields that form has.
Validation and other logic is not implemented, but this gives a
baseline from which components may be factored out.

awong-dev commented:
TBR=@akainic

Pre-emptively merging since this CL is pretty much completely copy-pasta so any sort of review is just a rubber-stamp.
